src/platforms/java/usage.mdx

```diff
@@ -255,6 +251,101 @@ public class MyClass {
 }
 ```
 
+```kotlin
+import io.sentry.HubAdapter
+import io.sentry.IHub
+import io.sentry.Scope
+import io.sentry.Sentry
+import io.sentry.SentryOptions
+import io.sentry.protocol.User
+
+class MyClass {
+    companion object {
+        private lateinit var hub: IHub
+    }
+
+    fun main(args: Array<String>) {
+        // NOTE: Replace the test DSN below with YOUR OWN DSN
+        Sentry.init { options: SentryOptions -> options.dsn = "https://f7f320d5c3a54709be7b28e0f2ca7081@sentry.io/1808954" }
+
+        // It is possible to go around the static Sentry API, which means
+        // you are responsible for making the IHub instance available
+        // to your code.
+        hub = HubAdapter.getInstance()
```

Comment:
to be honest I don't like leaking `HubAdapter` to the docs, I'd either revisit the API or find another alternative, this was meant to improve testability.
maybe @bruno-garcia

Reply:
This was a port from the legacy docs. I am also fine with removing it.

Reply:
Agreed! Changed to `new Hub(..)`.